sdcm/nemesis.py

```diff
@@ -4764,7 +4798,9 @@ def finalizer(exc_type, *_):
             context_manager.push(finalizer)
 
             with temporary_replication_strategy_setter(node) as replication_strategy_setter:
+                self._with_action_scope("Start add new node in new DC", target=self.target_node.name)
```

Comment:
I would move this message inside `self._add_new_node_in_new_dc()`

Reply:
it would look ugly - too much indentation. Anyway this one required fixing